addons/mail/static/src/core/web/chatter.js

```diff
@@ -105,6 +107,7 @@ export class Chatter extends Component {
         this.scrollPosition = useScrollPosition("root", undefined, "top");
         this.rootRef = useRef("root");
         this.onScrollDebounced = useThrottleForAnimation(this.onScroll);
+        this.popover = usePopover(FollowersList);
```

Comment:
could specify something like `followersPopover` or so since chatter is full of popovers.